docs/docs.go

```diff
@@ -3785,70 +3785,6 @@ const docTemplate = `{
                 }
             }
         },
-        "models.FeeUnit": {
-            "type": "object",
-            "properties": {
-                "bytes": {
-                    "description": "Bytes is a fee unit bytes representation.",
-                    "type": "integer",
-                    "example": 1000
-                },
-                "satoshis": {
-                    "description": "Satoshis is a fee unit satoshis amount.",
-                    "type": "integer",
-                    "example": 1
-                }
-            }
-        },
-        "models.MapProtocol": {
-            "type": "object",
-            "properties": {
-                "app": {
-                    "description": "App is a map protocol app.",
-                    "type": "string"
-                },
-                "keys": {
-                    "description": "Keys is a map protocol keys.",
-                    "type": "object",
-                    "additionalProperties": true
-                },
-                "type": {
-                    "description": "Type is a map protocol type.",
-                    "type": "string"
-                }
-            }
-        },
-        "models.OpReturn": {
-            "type": "object",
-            "properties": {
-                "hex": {
-                    "description": "Hex is a full hex of op return.",
-                    "type": "string"
-                },
-                "hex_parts": {
-                    "description": "HexParts is a slice of splitted hex parts.",
-                    "type": "array",
-                    "items": {
-                        "type": "string"
-                    }
-                },
-                "map": {
-                    "description": "Map is a pointer to a map protocol object.",
-                    "allOf": [
-                        {
-                            "$ref": "#/definitions/models.MapProtocol"
-                        }
-                    ]
-                },
-                "string_parts": {
-                    "description": "StringParts is a slice of string parts.",
-                    "type": "array",
-                    "items": {
-                        "type": "string"
-                    }
-                }
-            }
-        },
```

Comment:
Please double check and make sure we want to actually remove these models (which seem not to be related to the `shared_config`.